iothub/device/iot-device-client/src/test/java/com/microsoft/azure/sdk/iot/device/transport/mqtt/E4KConnectionMessagePayloadTest.java

```diff
@@ -0,0 +1,62 @@
+package com.microsoft.azure.sdk.iot.device.transport.mqtt;
+
+import com.google.gson.Gson;
+import com.microsoft.azure.sdk.iot.device.transport.mqtt5.E4KConnectionMessagePayload;
+import com.microsoft.azure.sdk.iot.device.transport.mqtt5.E4KConnectionState;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class E4KConnectionMessagePayloadTest
+{
+    @Test
+    public void TestSerialization()
+    {
+        String expectedDeviceClientType = "some device client type";
+        String expectedMqttVersion = "3.1.1";
```

Comment:
I am wondering if we should tie this to v5 only now that E4K requires Mqtt 5?

Reply:
Yes